helpers/misk.sh

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2019 Denis Gavrilyuk <karpa4o4@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+compress_with_bzip2() {
+    ${PYTHON} "${PIEMAN_UTILS_DIR}"/compress_image.py "bzip2" "${IMAGE}"
+}
+
+compress_with_xz() {
+    ${PYTHON} "${PIEMAN_UTILS_DIR}"/compress_image.py "xz" "${IMAGE}"
+}
+
+wget() {
+    ${PYTHON} "${PIEMAN_UTILS_DIR}"/wget.py "${@}"
+}
+
+uuidgen() {
+    echo "$(${PYTHON} -c "import uuid; print(uuid.uuid4())" 2>&1)"
```

Comment:
1. you don't have to redirect stderr to stdout here;
2. you don't even have to `echo`ing the result string.